starwars/src/components/PoppyCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './StarWars.css';
```

Comment:
It might be best to use a puppy-related naming convention for the puppycard CSS hehe.